CHANGELOG.md

```diff
@@ -27,17 +30,24 @@ All notable changes to this project will be documented in this file.
   - Improved naming for AMQP binding queues
   - Shelve VMs
   - Neutron Networking
+  - No longer using adminURL
 - Foreman: Exposed additional properties to reporting
 - Azure
   - Initial work for Inventory Collection, OAuth2, [azure-armrest gem](https://github.com/ManageIQ/azure-armrest)
   - Azure Provider models
   - Power Operations
+  - Added as a type for Cloud Provider discovery
+  - User Image Inventory
+  - Provision from private user image with template
 - RHEVM: Reconfigure of Memory and CPU
 - Orchestration: Reconfiguration
 - Reporting on Providers
   - Ability to report on Performance
   - Host Socket and Total VMs metrics
   - Watermark reports available out-of-the-box
+- Google
+  - Community addition
+  - Work has begun on this provider  
```

Comment:
This is weird from a CHANGELOG perspective...sounds more like a slide ;)  @blomquisg Thoughts here.